### ISSUE TITLE
Not maintained anymore?

### ISSUE THREAD

**Issue by jeluard** [danneu/klobbdown#1]
Hi!

Looks like this project is not maintained anymore. Do you mind if I update it and publish it somewhere else with the same EPL license?

**Comment by danneu** [comment]
Yeah, it's unmaintained weekendware. You can do as you like with it.

**Comment by hindol** [comment]
This project has a lot of things that might be of use to me. Any chance either one of you want to revive this? @danneu @jeluard

**Comment by jeluard** [comment]
Sorry I moved on to others projects :)
